### PR TITLE
actions overview/concepts page

### PR DESCRIPTION
**Pull request by johnbhiggins7**
### Description
- Added Actions Overview/Concepts page

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-XXX  

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Removed outdated references

### CODE REVIEW

File: docs/resources/action-flow-editor/overview.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+title: Overview
+tags: []
+toc_max_heading_level: 5
+---
+Effectively managing user interactions is essential for developing interactive applications. Designing interactivity involves two steps:
+
+1. Listening for Interaction (**Triggers**)
+2. Responding to Interaction (**Actions**)
+
+Both Triggers and Actions are just functions. Triggers are functions executed when a user performs a specific interaction, while Actions are functions executed in response to the triggered event. Common triggers are:
```
Comment (leighajarett) [inline docs/resources/action-flow-editor/overview.md:12]:
```suggestion
Triggers represent a specific event, while Actions are functions executed in response to the triggered event. Common triggers are:
```

I know triggers are technically callback functions, but I think that might be a bit more detail than is necessary?

File: docs/resources/action-flow-editor/overview.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+- Tapping on a button (**On Tap**)
+- Selecting an option from a dropdown list (**On Selected**)
+- Loading a page (**On Page Load**)
+
+Actions are the functions that are executed after the trigger has registered an event.
+
+## Action Flow Editor
+
+The Action Flow Editor is a visual, node-based editor used to define the functions for Triggers and Actions. This editor simplifies the process of creating and managing business logic.
```
Comment (leighajarett) [inline docs/resources/action-flow-editor/overview.md:22]: "define the functions for Triggers and Actions" --> "configure the functions that run in response to a trigger"?

**Comment by PoojaB26** [comment]
Hey @johnbhiggins7 Can we have all the changes mentioned in this [ticket](https://linear.app/flutterflow/issue/DEVR-315/user-interactivity-actions) and then we can review the changes together? It helps us know if we need to move things around to different sections and how the section looks in overall. Can we make this PR a draft till all the changes are available in this PR? Thanks.

File: docs/resources/action-flow-editor/overview.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
+---
+slug: overview
+title: Overview
+tags: []
```
Comment (leighajarett) [inline docs/resources/action-flow-editor/overview.md:4]: Might be useful to add something about action output variables?

**Review by leighajarett — CHANGES_REQUESTED** [review]
Few small comments

**Pull request closed without merging**